tests/pr_test.go

```diff
@@ -189,8 +190,9 @@ func TestAgentsSolutionInSchematics(t *testing.T) {
 			{Name: "log_analysis_instance_region", Value: region, DataType: "string"},
```

Comment:
`log_analysis_instance_region` can be removed now - we should no longer provision any new instances